app/client/components/navBar/styles.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
```

Comment:
better separation and reuse but eventually we should decide if we want modules or styled